src/supplies/supplies.controller.ts

```diff
@@ -55,4 +60,37 @@ export class SuppliesController {
       throw new HttpException('Cannot create Supply', HttpStatus.BAD_REQUEST);
     }
   }
+
+  @SetMetadata(MetadataKey.ALLOWED_ROLES, [UserRole.ADMIN])
+  @UseGuards(AuthGuard)
+  @Patch('/:id')
```

Comment:
PLease add `@ApiResponse({ type: Supply })` decorator so it is documented in swagger :)